src/components/Dashboard/ThisWeek/NewsletterMenu/MenuItems/ExportPhysical/ExportPhysical.tsx

```diff
@@ -41,105 +48,175 @@ const LabelButton = styled(Button)(({ theme }) => ({
   margin: theme.spacing(1, 0),
 }));
 
+const SelectInputLabel = styled(InputLabel)(({ theme }) => ({
+  fontWeight: 'bold',
+  margin: theme.spacing(2, 0, 0, 0),
+}));
+
+const ExportSelect = styled(Select)(() => ({
+  width: '100%',
+}));
+
 const ExportPhysical: React.FC<Props> = ({
   handleClose,
   accountListId,
 }: Props) => {
   const { t } = useTranslation();
   const [session] = useSession();
+  const [isExportingPdf, changeIsExportingPdf] = useState(false);
```

Comment:
If the user clicks ```PDF of Mail Merged Labels``` a new menu pops up to select the different filters they wish to apply to the export. Instead of creating a whole new component, I figured using some conditional logic inside this component would be simple enough. Let me know what you think though.

Reply:
That's fine. Could be worth making some sort of modal component that would let you pass these sections in as props:
```ts
<Modal
      title="Something"
      body={
        <SomeJSX />
      }
      footer={
        <SomeJSX />
      } />
```
I'm not sure what's best. This is okay.